### DISCUSSION TITLE
Single click to open files and folders

### DISCUSSION THREAD

**Discussion by j-pekka** [GiorgosXou/TUIFIManager#122] [Q&A] [answered]
By default you need to double click to open a folder. 
A lot of GUI file managers have an option where you only need a single click to open files and folders.
Is this behavior implemented? If so, how would I set this up?

**Comment by GiorgosXou** [comment]
> A lot of GUI file managers have an option where you only need a single click to open files and folders.

Interesting, I've only seen this on touch-devices.

> Is this behavior implemented? 

 Such a behavior is implemented for Termux exclusivly. Although, with a few tweaks it should work outside termux, too.
 
 > If so, how would I set this up?
 
There's no such option yet. But may I ask, where is this behavior intented to be used? Is it with another app like termux or just on a PC?

**Reply by j-pekka** [reply]
> Interesting, I've only seen this on touch-devices.
>
Atleast on Linux most of the common GUI file managers have this functionality. 
GnomeFiles(Nautilus), Nemo and Thunar have it as an option. Dolphin even has folders opening with a single click as the default.


> There's no such option yet. But may I ask, where is this behavior intented to be used? Is it with another app like termux or just on a PC?
>
Just on PC! 
I would love to have the option since I personally find it a lot more faster and comfortable.

**Reply by GiorgosXou** [reply, marked as answer]
Just for you https://github.com/GiorgosXou/TUIFIManager/commit/7adcc2f03f7d94ca7e4a64c11735a815f22ed41e :D
